src/components/VoteButton.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import { voteReview } from '../api.js'
+
+export function VoteButton({ reviewID, setReview, setDisplayAlert }) {
+  function vote(incVote) {
```

Comment:
I would make this function name a little bit more descriptive of what the function is meant to do.